src/components/new-request/name-input.vue

```diff
@@ -1,35 +1,23 @@
 <template>
-  <v-col cols="12">
-    <v-text-field :error-messages="message"
-                  @input="clearErrors()"
-                  @keydown.enter="handleSubmit"
-                  autocomplete="chrome-off"
-                  filled
-                  id="name-input-text-field"
-                  ref="nameInput"
-                  :rules="(searchValue && isMrasSearch) ? mrasRules : []"
-                  :label="isReadOnly ? '' : nameLabel"
-                  :class="{ 'read-only-mode': isReadOnly }"
-                  :disabled="isReadOnly"
-                  v-model="searchValue">
-      <template v-slot:append v-if="!isReadOnly">
-        <v-tooltip bottom
-                   content-class="bottom-tooltip search-tooltip"
-                   transition="fade-transition"
-                   :disabled="!isSearchAgain">
-          <template v-slot:activator="scope">
-            <v-icon class="name-search-icon"
-                    id="name-input-icon"
-                    color="primary"
-                    v-on="scope.on"
-                    :disabled="!isCorpNumValid"
-                    @click="startAnalyzeName">mdi-magnify</v-icon>
-          </template>
-          Search Again
-        </v-tooltip>
-      </template>
-    </v-text-field>
-  </v-col>
+  <v-row no-gutters>
+    <v-col cols="12" class="pa-0">
+      <v-text-field :error-messages="message"
+                    @input="clearErrors()"
+                    @keydown.enter="handleSubmit"
+                    autocomplete="chrome-off"
+                    :filled="!isReadOnly"
+                    id="name-input-text-field"
```

Comment:
Will this component be instantiated only once at a time? If not then this will create a duplicate ID browser warning; in that case use a classname instead.

Reply:
hmm okay good point will update

Reply:
Future?